src/main/java/org/jooq/lambda/Seq.java

```diff
@@ -3397,6 +3397,30 @@ public R next() {
     }
 
     /**
+     * Chunk a Stream based on a predicate
+     * <p>
+     * <code><pre>
+     * // ((1,1,2), (2), (3,4))
+     * Seq.of(1, 1, 2, 2, 3, 4).chunked(n -> n %2 == 0)
```

Comment:
We could always ignore `BOTH` for now and easily add it later if requested since it will be an enum.